data/NV/incentives.json

```diff
@@ -0,0 +1,1712 @@
+      "number": 350,
+      "unit": "ton"
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Offered to Plumas-Sierra Rural Electric Co-op residential members for the purchase and installation of a new qualifying Air Source Heat Pump."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "NV-72",
+    "authority_type": "utility",
+    "authority": "nv-plumas-sierra-rural-electric-cooperative",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "nv_plumas-SierraRuralElectricCooperative_rebates",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 250,
+      "unit": "ton"
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Offered to Plumas-Sierra Rural Electric Co-op residential members for the purchase and installation of a new qualifying Air Source Heat Pump."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "NV-73",
+    "authority_type": "utility",
+    "authority": "nv-plumas-sierra-rural-electric-cooperative",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "geothermal_heating_installation",
+    "program": "nv_plumas-SierraRuralElectricCooperative_rebates",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 600,
+      "unit": "ton"
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $600 per ton rebate for the purchase and installation of a new ENERGY STAR® Certified Ground Source Heat Pump."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "NV-74",
+    "authority_type": "other",
+    "authority": "nv-csa-reno",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "weatherization",
+    "program": "nv_cSAReno_weatherizationAssistanceProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 12666
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "The Weatherization Assistance Program implements home energy conservation measures at no cost for residents below 200% of the federal poverty line."
+    },
+    "low_income": "default"
+  },
+  {
+    "id": "NV-75",
+    "authority_type": "other",
+    "authority": "nv-rural-nevada-development-corporation",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "weatherization",
+    "program": "nv_ruralNevadaDevelopmentCorporation_weatherizationAssistanceProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 12666
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "The Weatherization Assistance Program implements home energy conservation measures at no cost for residents below 200% of the federal poverty line."
+    },
+    "low_income": "default"
```

Comment:
This doesn't seem right either; the webpage says 150% of the federal poverty line, and I don't see a reference to the state thresholds?